scheduler/scheduler-server/src/test/java/functionaltests/policy/license/TestLicensePolicy.java

```diff
@@ -41,7 +41,44 @@
 
 public class TestLicensePolicy extends SchedulerFunctionalTestLicensePolicy {
 
-    private static URL jobDescriptor = TestLicensePolicy.class.getResource("/functionaltests/descriptors/Job_simple_license_policy.xml");
+    private static URL JobSimpleJobLicensePolicy = TestLicensePolicy.class.getResource("/functionaltests/descriptors/Job_simple_job_license_policy.xml");
+
+    private static URL JobSimpleTaskLicensePolicy = TestLicensePolicy.class.getResource("/functionaltests/descriptors/Job_simple_task_license_policy.xml");
+
+    /**
+     * Tests that two independent jobs do not run at the same time, due to license limitation.
+     *
+     * @throws Exception
+     */
+
+    @Test
+    public void testLicensePolicyAtJobLevel() throws Throwable {
+
+        JobId jobId0 = schedulerHelper.submitJob(new File(JobSimpleJobLicensePolicy.toURI()).getAbsolutePath());
+        JobId jobId1 = schedulerHelper.submitJob(new File(JobSimpleJobLicensePolicy.toURI()).getAbsolutePath());
+
+        log("Waiting for jobs finished");
+        schedulerHelper.waitForEventJobFinished(jobId0);
+        schedulerHelper.waitForEventJobFinished(jobId1);
+
+        Scheduler scheduler = schedulerHelper.getSchedulerInterface();
+
+        while (true) {
```

Comment:
what is this testing ?

Reply:
i forgot to delete it.

Reply:
Finally following our discussion, we will keep it.